src/blocks/HeaderBlock/index.tsx

```diff
@@ -0,0 +1,183 @@
+import { colors } from '@hedviginsurance/brand'
+import { Container } from 'constate'
+import * as React from 'react'
+import styled from 'react-emotion'
+import { ContentWrapper, ErrorBlock } from '../../components/blockHelpers'
+import { ButtonLink } from '../../components/buttons'
+import { Togglable } from '../../components/containers/Togglable'
+import { HedvigWordmark } from '../../components/icons/HedvigWordmark'
+import { GlobalStory } from '../../storyblok/StoryContainer'
+import { getStoryblokLinkUrl } from '../../utils/storyblok-link'
+import { BaseBlockProps } from '../BaseBlockProps'
+import { Burger, TABLET_BP_DOWN } from './mobile'
+
+export const WRAPPER_HEIGHT = '5rem'
+export const HEADER_VERTICAL_PADDING = '1.5rem'
+export const TOGGLE_TRANSITION_TIME = 250
+
+const Wrapper = styled('div')(
+  ({ inverse, open }: { inverse: boolean; open: boolean }) => ({
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    right: 0,
+    zIndex: 100,
+    color: inverse ? colors.WHITE : undefined,
+
+    [TABLET_BP_DOWN]: {
+      bottom: open ? 0 : undefined,
+    },
+  }),
+)
+const Filler = styled('div')({
+  height: WRAPPER_HEIGHT,
+})
+const HeaderBackgroundFiller = styled('div')({
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  right: 0,
+  zIndex: -1,
+  height: WRAPPER_HEIGHT,
+  backgroundColor: colors.WHITE,
+  boxShadow: '0px 1px 2px rgba(0, 0, 0, 0.1), 0px 2px 5px rgba(0, 0, 0, 0.1)',
+})
+
+const InnerHeaderWrapper = styled('div')({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  width: '100%',
+  height: WRAPPER_HEIGHT,
+  padding: HEADER_VERTICAL_PADDING + ' 0',
+})
+const Menu = styled('div')(({ open }: { open: boolean }) => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+
+  [TABLET_BP_DOWN]: {
+    position: 'absolute',
+    display: 'block',
+    zIndex: 101,
+    width: '80%',
+    top: 0,
+    bottom: 0,
+    left: open ? '20%' : '100%',
+    paddingTop: `calc(${WRAPPER_HEIGHT} + ${HEADER_VERTICAL_PADDING})`,
+    background: colors.WHITE,
+    transition: `left ${TOGGLE_TRANSITION_TIME}ms`,
+    color: colors.OFF_BLACK_DARK,
+  },
+}))
+
+const MenuLink = styled('a')({
+  color: 'inherit',
+  textDecoration: 'none',
+  padding: '0 1rem',
+
+  '&:last-of-type': {
+    paddingRight: 0,
+  },
+
+  [TABLET_BP_DOWN]: {
+    display: 'block',
+    width: '100%',
+    padding: `1rem 2rem`,
+
+    '&:first-of-type': {
+      paddingTop: 0,
+    },
+  },
+})
+
+const LogoLink = styled('a')({
+  display: 'inline-flex',
+  paddingTop: 3, // fix to push down logo a little or it looks unbalanced
```

Comment:
no comments 🙅‍♂️

Reply:
I don't agree, i think we should _avoid_ comments but i don't think we should ban them completely. Sometimes they make sense. I would have to name this something like `PUSH__DOWN_LOGO_TO_AVOID_UNBALANCED_LOOK` which is just weird imo

Reply:
(ignoring this after irl discussion :) )